geth/jail/jail.go

```diff
@@ -119,15 +119,23 @@ func (jail *Jail) Call(chatID string, path string, args string) string {
 
 	// Due to the new timer assigned we need to clone existing cell to allow
 	// unique cell runtime and eventloop context.
-	cellCopy, err := cell.Copy()
-	if err != nil {
-		return makeError(err.Error())
-	}
+	// cellCopy, err := cell.Copy()
+	// if err != nil {
+	// 	return makeError(err.Error())
+	// }
 
 	// isolate VM to allow concurrent access
-	vm := cellCopy.CellVM()
+	vm := cell.CellVM()
 	res, err := vm.Call("call", nil, path, args)
 
+	// Due to the new event loop provided by ottoext.
+	// We need to ensure that all possible calls to internal setIntervals/SetTimeouts/SetImmediate
+	// work by lunching the loop.Run() method.
+	// Needs to be done in a go-routine.
+	// TODO(influx6): Figure out if we should do a throw Exception for returned error
+	// from loop.Run() call.
+	go cell.CellLoop().Run()
```

Comment:
Oh not at all. Even cell has only one loop per vm, the loop exists already has part of the cell, but just called within the `Call` method.

See: https://github.com/status-im/status-go/blob/2fba2b45eabc948f919b6944d14c643b76629c79/geth/jail/jail.go#L52
See: https://github.com/status-im/status-go/blob/2fba2b45eabc948f919b6944d14c643b76629c79/geth/jail/jail_cell.go#L34-L52

_Unless I misunderstood your statement_